src/pages/OnBoardingContent/OnBoardingContent.js

```diff
@@ -136,9 +168,23 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
+const steps = [1, 2, 3, 4, 5, 6];
```

Comment:
Não gosto muito desta solução visto q se outra pessoa quiser adicionar conteudo vai ter alguns problemas em decifrar q tem q alterar este array tb.. 

Não seria melhor criar uma function q retorna os vários steps e depois fazer algo como `React.Children.count(this.props.children)`? Isto dava-te automaticamente o `length` q precisas.

Reply:
acho que fiz o que pediste :) valida sff